_data/form.yml

```diff
@@ -0,0 +1,831 @@
+    type: address
+    address-fields:
+    - 'street'
+    - 'zip'
+
+  - label: Present use
+
+  - label: Proposed use
+
+- title: Construction work
+  fields:
+
+  - label: Please describe the work you are proposing in detail.
+    type: textarea
+    helptext: |
+      You should include:
+
+      - The number of ADUs you plan to add
+      - The city ordinances they each comply with
+      - A detailed description of all other planned work
+      - Any special authorizations or changes to the Planning Code or Zoning Maps
+
+  - label: Type of construction
+    type: select
+    options:
+    - label: "I-A: Fire Resistive Non-Combustible"
+    - label: "I-B: Fire Resistive Non-Combustible"
+    - label: "II-A: Protected Non-Combustible"
+    - label: "II-B: Unprotected Non-Combustible"
+    - label: "III-A: Protected Combustible"
+    - label: "III-B: Unprotected Combustible"
+    - label: "IV: Heavy Timber"
+    - label: "V-A: Protected Wood Frame"
+    - label: "V-B: Unprotected Wood Frame"
+
+  - label: Issued / estimated job cost
+    type: price
+
+  - label: How many months do you estimate construction will last?
+    type: number
+
+- title: Construction impact
+  fields:
+
+  - label: Area of excavation / disturbance
+    type: number
+    unit: square feet
+
+  - label: Amount of excavation
+    type: number
+    unit: cubic yards
+
+  - label: Including foundation work, how deep do you plan to disturb the surface below grade?
+    type: number
+    unit: feet
+
+  - label: Will the work involve any of the following?
+    type: checkbox
+    options:
+    - label: Adding height to the building
+      shows: new-height
+
+    - label: Adding a deck or horizontal extension
+      shows: new-ground-floor
+
+    - label: Repairing or replacing the building's foundation
+
+
+  - group: new-height
+    fields:
+    - label: State the new height at the center line of the front of the building.
+      helptext: If the height will be over 40 feet, you may need a shadow analysis.
+
+  - group: new-ground-floor
+    fields:
+    - label: What will the new ground floor area be?
+      type: number
+      unit: square feet
+
+- title: How are you currently using the land?
+  fields:
+  - label: Current parking space
+    type: number
+    unit: gross square feet
+
+  - label: Current residential space
+    type: number
+    unit: gross square feet
+
+  - label: Current retail commercial space
+    type: number
+    unit: gross square feet
+
+  - label: Current number of affordable dwelling units
+    type: number
+
+  - label: Current number of market-rate dwelling units
+    type: number
+
+  - label: Current number of buildings
+    type: number
+
+  - label: Current number of stories
+    type: number
+
+  - label: Current number of basements or cellars
+    type: number
+
+  - label: Current number of parking spaces
+    type: number
+
+  - label: Current number of bicycle spaces
+    type: number
+
+- title: What are you planning to change?
```

Comment:
Persis: Break this up into 10 pages, one for each field and group current / proposed together

Reply:
Less concerned about many pages than the possibility you'll forget content that was on the last page